templates/jupyter-error.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <style>
+    body,
+    h3 {
+      background-color: #fafbfc;
+      color: #333f52;
+      font-family: "Montserrat", sans-serif;
+      font-feature-settings: "tnum";
+      font-variant-numeric: tabular-nums;
+      -webkit-font-smoothing: antialiased;
+    }
+
+    p {
+      font-size: 15px;
+    }
+  </style>
+  <h3>
+    Terra’s Preview Service has encountered an error rendering this file.
+  </h3>
+</head>
+
+<body>
+  <p>
+    <strong>Message:</strong> {{error}}
+  </p>
+
+  <p>
+    To generate a preview file from within Jupyter, you can use <strong>File -> Print Preview</strong>, or <strong>
```

Comment:
this might change once we migrate to jupyterlab..just sth to keep in mind in the future I guess